### PR TITLE
Kickstart data onboarding 

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds docs and release notes for sample data placement in apps/dashboards for trial users.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-474

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208394450088124
  - https://app.asana.com/0/0/1208394448610552

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, Pawan Murthy reviewed the draft in a [Google Doc](https://docs.google.com/document/d/1bZ0JVyY9O_oNVT2DNjd_TYeb78iCsR9Ehd26xqmOfhA/edit?tab=t.0) and approved it. See his comment in Asana: https://app.asana.com/0/0/1208394450088124/1208670600698786/f

File: docs/get-started/quickstart.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ OpenTelemetry is our preferred collector unless you're configuring security data
 #### Installed and Hosted Collector
 
 1. Log in to Sumo Logic.
-1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. 
+1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**.
```
Comment (kimsauce) [inline docs/get-started/quickstart.md:52]:
```suggestion
1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Collection > Collection**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Configuration**, and then under **Data Collection** select **Collection**. You can also click the **Go To...** menu at the top of the screen and select **Collection**. 
```

File: docs/get-started/quickstart.md [hunk 2]
```diff
@@ -25,6 +25,33 @@ What you'll learn:
...
+
+* **Immediate insights**. Explore Sumo Logic right away, without initial data setup, to quickly see its value and decide if it’s a fit for you.
+* **Quick setup**. Placeholder data removes technical hurdles, making onboarding faster and easier.
+* **Guided experience**. Access pre-built dashboards and reports that demonstrate real-world scenarios, helping you make the most of your trial.
+* **Safe exploration**. Evaluate our platform in a secure environment with no exposure of sensitive data.
+
+### How it works
+
+1. **User assignment**. When you start your trial, you will be automatically assigned to either the placeholder data workflow or the custom data workflow, based on your user profile.
+2. **Preloaded sample data**. If you are assigned the placeholder data option, sample data will be preloaded into the platform, along with dashboards and log searches tailored to your use cases, such as Monitoring & Troubleshooting (M&T) or Compliance & Security (CIS).
```
Comment (jpipkin1) [inline docs/get-started/quickstart.md:42]: @kimsauce, I'd remove this phrase: 
", such as Monitoring & Troubleshooting (M&T) or Compliance & Security (CIS)" 

I'm not sure those terms are correct. In Sumo Logic, "CIS" stands for "Cloud Information Security", not "Compliance & Security".

**Review by jpipkin1 — APPROVED** [review]

File: docs/get-started/quickstart.md (hunk 2 above)
Comment (kimsauce) [inline docs/get-started/quickstart.md:42]:
```suggestion
2. **Preloaded sample data**. If you are assigned the placeholder data option, sample data will be preloaded into the platform, along with dashboards and log searches tailored to your use cases, such as monitoring, troubleshooting, or security.
```

**Pull request merged**